app/managers/weaviate_manager.py

```diff
@@ -643,11 +651,20 @@ def update_sample_question(self, sample_question: DatabaseSampleQuestion):
 
     def delete_sample_question(self, id: str):
```

Comment:
kann man die methode nicht weglassen und immer mit ner ids list callen oder in der methode die andere aufrufen und die id in ne liste packen?